packages/hurl/src/cli/options/commands.rs

```diff
@@ -272,6 +272,16 @@ pub fn json() -> clap::Arg {
         .action(clap::ArgAction::SetTrue)
 }
 
+pub fn max_filesize() -> clap::Arg {
+    clap::Arg::new("max_filesize")
+        .long("max-filesize")
+        .value_name("NUM")
+        .default_value("0")
```

Comment:
Don't use a default value of 0, we'll use an Option<usize> to model max_filesize